projects/elements/src/lib/lazy-elements/lazy-element/lazy-element.directive.spec.ts

```diff
@@ -1,7 +1,21 @@
-import { Component, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+import { Component, CUSTOM_ELEMENTS_SCHEMA, NgModule } from '@angular/core';
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
 
 import { LazyElementDirective } from './lazy-element.directive';
+import { LazyElementsLoaderService } from '../lazy-elements-loader.service';
+
+@Component({
+  template: `
+    <p class="loading">Spinner...</p>
+  `
+})
+class SpinnerTestComponent {}
```

Comment:
Is this needed? We have a real Spinner component which does the same ?

Reply:
This is in `elements` lib, and the real Spinner Component is in `elements-demo` app